src/options.coffee

```diff
@@ -131,4 +131,8 @@ options =
   version:
     description: "Show version number.\n"
 
+  'hook-timeout':
```

Comment:
I'm not sure if the `hook-timeout` configuration parameter isn't confusing because it applies to the `HookWorkerClient` only and it doesn't apply for native Node.js hooks.

I suggest:
a) to make it work for the native Node.js hooks as well
b) rename it  ti something` like: `hook-worker-timeout`

Option a) makes more sense in my opinion